packages/app-content-pages/src/screens/Teams/Teams.js

```diff
@@ -16,18 +16,9 @@ import {
   StyledHeading
 } from '../../shared/components/SharedStyledComponents/SharedStyledComponents.js'
 
-const isBrowser = typeof window !== 'undefined' // to handle testing environment
-
 function TeamComponent({ teamData = [], sections = [] }) {
   const { t } = useTranslation('components')
-  const [activeSection, setActiveSection] = useState('')
-
-  useEffect(function onMount() {
-    const slug = isBrowser ? window.location.hash.slice(1) : ''
-    setActiveSection(slug)
-  }, [])
-
-  const sectionsPlusAll = [{ name: t('Sidebar.all'), slug: '' }, ...sections]
```

Comment:
I decided to remove the "All" option in the Teams sidebar because the Alder Planetarium section heading is basically at the top of the viewport, just below the Publications H1. In contrast to the Publications page which as a paragraph of help text at the top of the page, it doesn't seem like there's any benefit to an "All" link in the sidebar here especially when highlight-on-scroll makes it obvious where you are in the page.